zerver/models/prereg_users.py

```diff
@@ -49,6 +49,14 @@ class PreregistrationRealm(models.Model):
         UserProfile, null=True, related_name="+", on_delete=models.SET_NULL
     )
 
+    IMPORT_FROM_CHOICES = {
+        "None": 100,
+        "Slack": 200,
+    }
```

Comment:
This might be better as a string field? I feel like "slack" would be easier to work with than a magic number.